docs/Development Concepts/4.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Transports in Waku
+---
+
+Waku v2 is built in top of libp2p, and like libp2p it strives to be transport agnostic. We define a set of recommended transports in order to achieve a baseline of interoperability between clients.
```

Comment:
I am guessing you got this section from the RFC. I'd make it more informal and just mention:

- Waku nodes uses TCP to communicate by default. A service node should be using TCP to connect to other nodes and listen to
- In environments where TCP is not available, such as browser, secure websocket is used. service nodes are encouraged to setup a SSL certificate to enable incoming connections from browser and serve them.
- Other protocols such as WebRTC, WebTransport and QUIC have been studied.

Also include link to issues. WebRTC and WebTRanspoirt are in js-waku repo. Not sure about QUIC, have a search in vacp2p orga.